src/gap.c

```diff
@@ -1278,13 +1277,6 @@ Obj FuncCALL_WITH_CATCH( Obj self, Obj func, Obj args )
       ErrorMayQuit("CALL_WITH_CATCH(<func>,<args>): <func> must be a function",0,0);
     if (!IS_LIST(args))
       ErrorMayQuit("CALL_WITH_CATCH(<func>,<args>): <args> must be a list",0,0);
-    if (!IS_PLIST(args))
-      {
-        plain_args = SHALLOW_COPY_OBJ(args);
-        PLAIN_LIST(plain_args);
-      }
-    else 
-      plain_args = args;
```

Comment:
So why was the above code removed? This change is not mentioned in the commit message. Why is it safe? (I am not saying it is, but it is surpring to me to find it mingled with the rest).